services/server/src/openapi.yaml

```diff
@@ -1,13 +1,30 @@
 openapi: 3.1.0
 info:
-  version: 1.0.0
-  title: Sourcify API
-  description: API to interact with Sourcify
+  version: 2.0.0
+  title: Sourcify APIv2
+  description: |-
+    Welcome to the Sourcify's APIv2.
+
+    Important differences between the deprecated [legacy API](https://docs.sourcify.dev/docs/api/) and the new APIv2:
+    - **Ticketing**: The verfication requests resolve into tickets/verification jobs. 
+      - Previously the verification happened during the HTTP request, which resulted in timeouts if compilation took longer
+    - **Standard JSON as default**: In the current design we take the standard JSON format as our main verification endpoint. We still support verification with metadata at `/v2/verify/metadata`.
+    - **Lean API**: We keep the number of endpoints minimal compared to v1. We won't have a session API. 
+    - **Detailed contract response**: Prev. we only returned contract files of a contract. Now we can return details at `/contract/{chainId}/{address}`.
   license:
     name: MIT
     url: https://github.com/ethereum/sourcify/blob/master/LICENSE
+  contact:
+    name: Sourcify
+    url: 'https://sourcify.dev'
+    email: hello@sourcify.dev
 servers:
-  $ref: "servers.yaml"
+  - url: https://sourcify.dev/server
+    description: Production server
+  - url: https://staging.sourcify.dev/server
+    description: Staging server
+  - url: http://localhost:5555
+    description: Local development server address on default port 5555.
```

Comment:
We can still use the `servers.yaml` file or remove it if we don't need it anymore

Reply:
I thought I deleted it, thanks